src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = this.random.nextInt(FigureTypes.values().length);
+        FigureTypes figureType = FigureTypes.values()[index];
+        String color = this.colorSupplier.getRandomColor();
+
+        switch (figureType) {
+            case CIRCLE:
+                return this.createCircle(
+                        color,
+                        this.random.nextDouble()
```

Comment:
Remove the input from all private methods, and just create random numbers and colors inside the private methods

Reply:
Done